packages/cli/create-strapi-app/.eslintignore

```diff
@@ -2,3 +2,4 @@ node_modules/
 .eslintrc.js
 dist/
 bin/
+templates/
```

Comment:
Why wouldn't we want to run eslint on the templates?

Reply:
because some are js files, some ts etc, they are not connected to the eslint config of the package itself